### DISCUSSION TITLE
Hided all fields from a block but empty block still showin

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#16669] [General]
Dependent fields allowing to hide fields on certain conditions, but the block still shows. How to fix it and maybe fix can be included in the core?

**Comment by mariuszkrzaczkowski** [comment]
show some example

**Comment by s2b-git** [comment]
I can see it too. Below you can see "Quotation Details" block. All fields are hidden by Dependent fields.. but the block itself still showing header
![image](https://user-images.githubusercontent.com/84110093/203066673-f193eca2-eb25-4b17-ac3d-3bfe346ff966.png)

**Reply by mariuszkrzaczkowski** [reply]
show field management panel

**Reply by s2b-git** [reply]
![image](https://user-images.githubusercontent.com/84110093/203145832-0b0f460b-19c5-446b-80f0-ac0cf1227471.png)
